starknet_py/utils/iterable.py

```diff
@@ -0,0 +1,12 @@
+from typing import TypeVar, Iterable, Union
+
+T = TypeVar("T")
+
+# pyright: reportGeneralTypeIssues=false
+def ensure_iterable(value: Union[T, Iterable[T]]) -> Iterable[T]:
```

Comment:
`Union[T, Iterable[T]]` it would make more sense to type it as `Any` because it is basically that.

Reply:
How would you type the return value without losing type in that case without providing generic type directly?

Reply:
I though maybe we could still use `Iterable[T]` as return type. In that case, we can keep it as it is.

Reply:
Python would have no way of knowing what the T should be there